services/provider/server/consumer.go

```diff
@@ -36,73 +30,35 @@ func newConsumerManager(ctx context.Context, cl client.Client, namespace string)
 		return nil, fmt.Errorf("failed to list storage consumers. %v", err)
 	}
 
-	nameByTicket := map[string]string{}
 	nameByUID := map[types.UID]string{}
 
 	for _, consumer := range consumers.Items {
 		nameByUID[consumer.UID] = consumer.Name
-
-		if ticket, ok := consumer.GetAnnotations()[TicketAnnotation]; ok {
-			nameByTicket[ticket] = consumer.Name
-		}
 	}
 
 	return &ocsConsumerManager{
-		client:       cl,
-		namespace:    namespace,
-		nameByTicket: nameByTicket,
-		nameByUID:    nameByUID,
+		client:    cl,
+		namespace: namespace,
+		nameByUID: nameByUID,
 	}, nil
 }
 
-// Create creates a new storageConsumer resource, updates the consumer cache and returns the storageConsumer UID
-func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageClientOnboarding, storageQuotaInGiB int) (string, error) {
```

Comment:
Is this the only function that does away? what about EnableStorageConsumer? Delete? AddAnnotation? RemoveAnnotation?

Reply:
didn't get you, what are those?

Reply:
fixed.